space-invaders.asm

```diff
@@ -1,12 +1,65 @@
 ; constants
-%define NUM_INVADERS 12
+
+; GAME WINDOW PARAMETERS
 %define GAME_WIDTH 27
 %define GAME_HEIGHT 25
 
+; GAME PARAMETERES
+%define NUM_INVADERS 12
 %define INVADERS_MOVE_CYCLES 40
 %define INVADERS_SHOOT_CYCLES 4
 %define BULLETS_MOVE_CYCLE 28
 
+; SPECIAL CONSTANT
+%define INVALID_STATE 0x0000
+
+; ICONS
+%define PLAYER 'M'
+%define INVADERS 'T'
+%define BULLET '|'
+%define EXPLOSION_BULLET '#'
+%define WALL '#'
+
+; GAME WINNERS
+%define PLAYER_WIN 0
+%define INVADERS_WIN 1
+
+; GAME STATES
+%define START_STATE 0
```

Comment:
As above, these should get a prefix like `PROGRAM_STATE_START`, `PROGRAM_STATE_GAME`, ...

Reply:
OK!